test/load/src/main.rs

```diff
@@ -65,12 +186,19 @@ async fn main() -> Result<(), GooseError> {
             scenario!("Basic (authed)")
                 .register_transaction(transaction!(get_index))
                 .register_transaction(transaction!(signup_login))
-                .register_transaction(transaction!(get_recommendations)),
+                .register_transaction(transaction!(get_recommendations))
+                .register_transaction(transaction!(get_listing))
+                .register_transaction(transaction!(create_listing))
+                .register_transaction(transaction!(create_review))
+                .register_transaction(transaction!(get_user))
+                .register_transaction(transaction!(get_message_threads))
+                .register_transaction(transaction!(search_listings))
+                .register_transaction(transaction!(get_search_history)),
```

Comment:
I'll add some for recommendations and listings, but I don't think multiple message gets make sense since they'll only get all of them once likely? Unless we want to do one at the start and one at the end.